config/src/cluster_config.rs

```diff
@@ -128,31 +125,42 @@ impl CheckConfig for ClusterConfig {
             });
         }
 
-        if let Err(e) = self.meta_service_addr.to_socket_addrs() {
-            ret.add_error(CheckConfigItemResult {
-                config: config_name.clone(),
-                item: "meta_service_addr".to_string(),
-                message: format!("Cannot resolve 'meta_service_addr': {}", e),
-            });
+        for meta_addr in self.meta_service_addr.iter() {
+            if let Err(e) = meta_addr.to_socket_addrs() {
+                ret.add_error(CheckConfigItemResult {
+                    config: config_name.clone(),
+                    item: meta_addr.clone(),
+                    message: format!("Cannot resolve 'meta_service_addr': {}", e),
+                });
+            }
         }
-        if let Err(e) = self.http_listen_addr.to_socket_addrs() {
+
+        let deafault_node_ip = "0.0.0.0".to_string();
+
+        let default_http_addr = deafault_node_ip.clone() + ":" + &self.http_listen_port.to_string();
+        if let Err(e) = default_http_addr.to_socket_addrs() {
```

Comment:
**Error** Variant `default_http_addr` is not the user defined network listener.

Reply:
I don't quite understand. HTTP should be provided to users by tskv for access